Set-WindowsProductKey/Set-WindowsProductKey.psm1

```diff
@@ -1,38 +1,44 @@
 ﻿<#
 .Synopsis
-   Set the Windows product key.
+    Set the Windows product key.
 .DESCRIPTION
-   Set the Windows product key on a local or remote computer.
+    Set the Windows product key on a local or remote computer.
 .EXAMPLE
-   Set-WindowsProductKey -ProductKey "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
-   This will set the local machines product key.
+    Set-WindowsProductKey -ProductKey "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
+    This will set the local machines product key.
 .EXAMPLE
-   Set-WindowsProductKey -ComputerName "bob-desktop" -ProductKey "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
-   This will set the product key on the computer bob-desktop.
+    Set-WindowsProductKey -ComputerName "bob-desktop" -ProductKey "XXXXX-XXXXX-XXXXX-XXXXX-XXXXX"
+    This will set the product key on the computer bob-desktop.
 #>
 function Set-WindowsProductKey
 {
     [CmdletBinding()]
     Param
     (
         # ComputerName
-        [string[]]$ComputerName="localhost",
+        [string[]]$ComputerName=$null,
```

Comment:
In some of the updated commandlets "localhost" has been left in.  Should I update all of them to read $null?